### PR TITLE
chore: update dependency open to v9

### PR DESCRIPTION
**Pull request by vagusX**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [x] Other (Update open dependency to version 9.0.0 and use dynamic import in print-changelog.js)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 71b670d</samp>

Updated the `open` module to the latest version and fixed the related import and type issues in the `print-changelog.js` and `TokenTable` files.

### <samp>🤖 Generated by Copilot at 71b670d</samp>

*  Update `open` dependency to version 9.0.0 and use dynamic import in `print-changelog.js` script ([link](https://github.com/ant-design/ant-design/pull/41765/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L249-R249), [link](https://github.com/ant-design/ant-design/pull/41765/files?diff=unified&w=0#diff-793cc8bdee7df8939c00696b7ba6d33cd9e2471f7882af6e96ca0535f9e2a11aL7), [link](https://github.com/ant-design/ant-design/pull/41765/files?diff=unified&w=0#diff-793cc8bdee7df8939c00696b7ba6d33cd9e2471f7882af6e96ca0535f9e2a11aR279-R280))
*  Remove unnecessary `any` type assertion from `value` property of `TokenTable` component in `.dumi/theme/builtins/TokenTable/index.tsx` ([link](https://github.com/ant-design/ant-design/pull/41765/files?diff=unified&w=0#diff-16fa19cd7384aaec1f2aa2733ad3a35affb9541febf0bd5aaa779eb0c7d9890aL107-R107))

### CODE REVIEW

File: .dumi/theme/builtins/TokenTable/index.tsx [hunk 1]
```diff
@@ -104,7 +104,7 @@ const TokenTable: FC<TokenTableProps> = ({ type }) => {
           name: token,
           desc: lang === 'cn' ? meta.desc : meta.descEn,
           type: meta.type,
-          value: (defaultToken as any)[token],
+          value: defaultToken[token],
```
Comment (li-jia-nan) [inline .dumi/theme/builtins/TokenTable/index.tsx:107]: 大佬这里还原一下吧，我在另一个 PR 里面改掉了

**Review by zombieJ — APPROVED** [review]

File: scripts/print-changelog.js [hunk 2]
```diff
@@ -277,6 +276,8 @@ async function printLog() {
   });
 
   console.log(chalk.green('Start changelog preview editor...'));
+  const { default: open } = await import('open');
```
Comment (afc163) [inline scripts/print-changelog.js:279]: 居然可以这样用

Comment (yoyo837) [reply scripts/print-changelog.js:279]: 应该有node 版本要求.

Comment (li-jia-nan) [reply scripts/print-changelog.js:279]: dynamic import ……在 CJS 下面用 ESM 的常见做法，我昨天倒是没想到这个

Comment (MadCcc) [reply scripts/print-changelog.js:279]: 改用 ts-node --esm 呢，把脚本改成 esm，现在项目里有 node 专用的 tsconfig.node.json

**Pull request merged**